### ISSUE TITLE
Bug: package.json does not have a valid "main" entry?

### ISSUE THREAD

**Issue by Mach12** [prettier/plugin-pug#211]
## Info

| Tool      | Version                 |
| --------- | ----------------------- |
| Plugin    | v1.15.0                  |
| Prettier  | v2.2.1                  |
| Framework | vue |
| Node      | v12.22.1                  |
| OS        | linux           |

## Prettier config

```json
{
  "trailingComma": "none",
  "tabWidth": 2,
  "semi": false,
  "singleQuote": true,
  "overrides": [
    {
      "files": "*.pug",
      "options": {
        "parser": "pug",
        "singleQuote": false
      }
    }
  ],
  "printWidth": 120
}
```

## Input

```pug
//- N/A
```

## Output or Error

```bash
 error  in ./src/main.ts

Syntax Error: Error: Cannot find module '<project_path>/node_modules/@prettier/plugin-pug/dist/index.js'. Please verify that the package.json has a valid "main" entry
Occurred while linting <project_path>/src/main.ts:1
    at Array.map (<anonymous>)
    at Array.map (<anonymous>)
    at Array.forEach (<anonymous>)


 @ multi (webpack)-dev-server/client?http://192.168.2.71:8080&sockPath=/sockjs-node (webpack)/hot/dev-server.js ./src/main.ts
```

## Expected Output

```pug
//- N/A
```

## Additional Context

Looking at the contents of `node_modules/@prettier/plugin-pug`, it would seem the entire `dist` directory is missing. It was not missing in the 1.14 releases, might be some kind of problem with deployment or something?

**Comment by Shinigami92** [comment]
I'm so sorry 😓 
I added a `prepublishOnly` script now so that shouldn't happen again
https://github.com/prettier/plugin-pug/commit/4b874d3a64098911760ae842cd2e3fc4e4bcf92b

**Comment by Shinigami92** [comment]
`index.js` is now available again in `1.15.1`
Checked: https://unpkg.com/@prettier/plugin-pug@1.15.1/dist/index.js
